components/network/sshfs/Makefile

```diff
@@ -9,22 +9,23 @@
 #
 
 #
-# Copyright 2016 Adam Stevko. All rights reserved.
+# Copyright 2017 Adam Stevko. All rights reserved.
```

Comment:
Extend to your years of activity on this component? Also to be seen by readers that it is not something that appeared just today :)